src/CoreBundle/private/modules/CMSModuleHelp/CMSModuleHelp.class.php

```diff
@@ -25,23 +29,37 @@ class CMSModuleHelp extends TCMSModelBase
     protected $oTableConf = null;
     protected $oTableList = null;
 
-    public function Init()
-    {
-    }
-
     public function &Execute()
     {
         $this->data = parent::Execute();
 
+        $translator = $this->getTranslator();
+
+        $this->data['translator'] = $translator;
```

Comment:
I'd advocate against passing service dependencies to views as it increases coupling between module and view.

Reply:
Right. Now setting the translated title in the module instead.